### DISCUSSION TITLE
Alih Bahasa ke Indonesia

### DISCUSSION THREAD

**Discussion by AaEzha** [nurfachmi/perpustakaan#14] [Ideas]
Saat ini menggunakan dua bahasa yang tercampur, bahasa Inggris dan Indonesia.

Saat ini juga, saya ingin mengubah seluruhnya menjadi berbahasa Indonesia. Projek ini dibuat oleh orang Indonesia dan untuk orang Indonesia kembali.

Mohon bantuannya.

---

## Update

Saat ini sudah diimplementasikan localization secara keseluruhan pada modul Buku. Yang belum diimplementasikan, antara lain:

- [ ] Modul Pengguna
- [ ] Modul Peran
- [ ] Modul Anggota
- [ ] Modul Peminjaman Buku

**Comment by akbarhlubis** [comment]
Maaf mas @AaEzha saya lumayan masih pemula soal ini, kira kira mengubahnya harus dengan variabel atau manual kah mas setiap teksnya? terima kasih

**Comment by AaEzha** [comment]
Lebih enak jika terstruktur menggunakan [localization](https://laravel.com/docs/10.x/localization#main-content) sehingga lebih mudah dalam pengembangan ke depannya, mas.

Saat ini masih manual.

**Comment by akbarhlubis** [comment]
Okay mas Noted, menarik juga soal **localization** ini, mungkin nanti saya coba kulik kulik gimana cara implementasinya.
